helm/slurm/templates/controller/controller-statefulset.yaml

```
@@ -182,7 +182,9 @@ spec:
         resources:
           requests:
             storage: {{ .Values.controller.persistence.size | quote }}
+        {{- if .Values.controller.persistence.storageClass }}
         storageClassName: {{ .Values.controller.persistence.storageClass | quote }}
+        {{- endif }}
```

FWIW this is a slightly cleaner way to do conditional templating in Helm:
```suggestion
        {{- with .Values.controller.persistence.storageClass }}
        storageClassName: {{ . | quote }}
        {{- end }}
```